regen-network/regen-web
# feat: link ecocredit table batch denom and credit class to respective…

… pages

## Description

Closes: regen-network/regen-registry#946

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...


### How to test
1. Go to https://deploy-preview-970--regen-registry.netlify.app/ecocredits/dashboard
2. Confirm `batch denom` and `credit class` link to their respective pages


### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...
